src/lib/files.js

```diff
@@ -46,11 +46,12 @@ export function normalizeFiles (files) {
  * @returns {Promise<FileDownload>}
  */
 async function downloadSingle (file, gatewayUrl, apiUrl) {
-  let url, filename, method
+  let url, filename, name, method
```

Comment:
No need to make `name`  a variable up here when you're only using it in line 53.